package/Dockerfile

```diff
@@ -84,7 +84,7 @@ ENV CATTLE_SYSTEM_AGENT_INSTALLER_IMAGE rancher/system-agent-installer-
 # make sure the ENV CATTLE_SYSTEM_AGENT_INSTALL_SCRIPT is consistent with pkg/settings/setting.go to utlize the local version of install script downloaded during build/package
 ENV CATTLE_SYSTEM_AGENT_INSTALL_SCRIPT ${CATTLE_SYSTEM_AGENT_DOWNLOAD_PREFIX}/${CATTLE_SYSTEM_AGENT_VERSION}/install.sh
 ENV CATTLE_SYSTEM_AGENT_UNINSTALL_SCRIPT ${CATTLE_SYSTEM_AGENT_DOWNLOAD_PREFIX}/${CATTLE_SYSTEM_AGENT_VERSION}/system-agent-uninstall.sh
-ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 104.0.0+up0.7.0
+ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 105.00
```

Comment:
Isn't this missing `.`?
`ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 105.0.0`

I guess this is not caught by CI as SUC is not used in any test cases which is somewhat of a gap for sure.

Reply:
Thank you for catching the error!
Here is the follow-up PR to correct the value: https://github.com/rancher/rancher/pull/47730